web/satellite/tests/unit/project/__snapshots__/EditProjectDetails.spec.ts.snap

```diff
@@ -103,13 +127,19 @@ exports[`EditProjectDetails.vue renders correctly 1`] = `
       <p class="project-details__wrapper__container__label">Name</p>
       <div class="project-details__wrapper__container__name-area">
         <p class="project-details__wrapper__container__name-area__name">test</p>
-        <div class="container white" style="width: 64px; height: 28px; border-radius: 6px; font-size: 16px;"> <span class="label">Edit</span></div>
+        <div class="container white" style="width: 64px; height: 28px; border-radius: 6px; font-size: 16px;">
+          <!---->
+          <!----> <span class="label">Edit</span>
+        </div>
       </div>
       <!---->
       <p class="project-details__wrapper__container__label">Description</p>
       <div class="project-details__wrapper__container__description-area">
         <p class="project-details__wrapper__container__description-area__description">test</p>
-        <div class="container white" style="width: 64px; height: 28px; border-radius: 6px; font-size: 16px;"> <span class="label">Edit</span></div>
+        <div class="container white" style="width: 64px; height: 28px; border-radius: 6px; font-size: 16px;">
+          <!---->
+          <!----> <span class="label">Edit</span>
```

Comment:
The extra comments are still present.

Reply:
Hi Egon, these are automatically generated during the snapshot updating process. When I ran a find/replace for them on PaymentMethods.spec.ts.snap they were automatically re-added at every blank space and empty line, so I went back in and removed them manually. There are 50-60 of these per snapshot so we may need to look into an better approach to removing these (or not generating them in the first place) moving forward.

Reply:
Sure, I didn't mean that the snaps need to be edited manually. We need to figure out why it generates them in the first place and see whether we can fix that.